runtime/src/main/java/org/corfudb/runtime/collections/TransactionPoller.java

```diff
@@ -69,18 +83,12 @@ private void pollTxStream() {
  
         long lastReadAddress = streamContexts.get(0).getLastReadAddress();
 
-        StreamOptions options = StreamOptions.builder()
-                .ignoreTrimmed(true)
-                .cacheEntries(false)
-                .build();
-        IStreamView txStream = runtime.getStreamsView()
-                .get(ObjectsView.TRANSACTION_STREAM_ID, options);
         log.trace("Seeking txStream to {}", lastReadAddress + 1);
-        txStream.seek(lastReadAddress + 1);
+        txnStream.seek(lastReadAddress + 1);
```

Comment:
Just make sure that when you seek, next will return lastReadAddress + 1 and not lastReadAddress + 2, otherwise you might lose an update.

Reply:
Just verify the seek semantics, I can't remember of the top of my head.